File: unit09-lab/ruby/Name.rb
# Name.rb Holds information about the Name class
#
# Completed by: Robert Lowman
# Date: 11/10/2016
#################################################

module Names
  class Name
    def initialize(first, middle, last)
      @first, @middle, @last = first, middle, last
    end
    attr_reader :first, :middle, :last

    def fullName()
      return first + " " + middle + " " + last
    end

    def print()
      puts( fullName )
      fullName()
    end
  end
end

File: unit09-lab/python/Name.py
## Name.py holds information about a first,
##     middle, and last name
##
## Completed by: Robert Lowman
## Date: 12/1/2016

class Name:

    def __init__(self, first, middle, last):
        self.first = first
        self.middle = middle
        self.last = last

    def fullName(self):
        return self.first + " " + self.middle + " " + self.last

    def printName(self):
        print self.fullName()
